book/website/communication/outreach/outreach-effectively.md

```diff
@@ -0,0 +1,104 @@
+
+## Visual Communication
+
+Visual communication provides a way to include actual data into your message.
+This can be done in a variety of media, ranging from simple graphics designed to illustrate a point to complex web-apps to explore and visualise data.
+Researchers often produce graphics that are good at illustrating a technical point but may not fully emphasize the main message. The goal of adapting graphics for communicating research to a wider audience is to remove any information that is not necessary to tell the main story.
+If you are trying to communicate your ideas to a journalist, it can be very helpful to have this in mind when crafting any graphics that you share with them, which is likely to help get your ideas across and tell a coherent and accurate story (example {cite}`Elliott2019viz`).
+Other avenues for visual communication include posters, slides, and multimedia such as videos and cartoons can be useful ways to add additional visual dimensions to how you communicate results.
+
+News articles that cover research results often derive visuals from the research article itself.
+In this case, many news outlets have illustrators in the house who create these visuals based on the originals in the paper.
+However, significant editing usually takes place before placing a graphic in a news article such that the original is modified in many ways.
+For some excellent examples of this, a [talk by a NY Times science illustrator](http://style.org/ku/) provides some insight into the variety of ways that different graphics have been adapted for use by the newspaper.
+Note however that the talk makes it crystal clear that one needs to define the audience prior to making any illustrations, and that his audience is NY Times readers.
+
+As with the discussion above, once the audience has been defined and the message that the illustration needs to convey has been set, the illustrator went through multiple iterations and solicited immediate feedback from editors and other non-specialists to hone the clarity of the presentation.
+Note that this is similar to the process of refining the oral or editing the written message as detailed above.
+In many cases, researchers often make a quick and dirty figure using the standard settings on a graphics package and do not go through extensive revisions to ensure that the figure is clear, not misleading, and contains no extraneous information that would detract from the main message.
+
+There is a range of ways to illustrate information visually.
+One idea of how to arrange this is from Scientific American Art Director Jen Christensen {cite}`Christiansen2021viz`.
+
+At one end of the spectrum, you have illustrations that are figurative and just meant to provide a picture of the content under question.
```

Comment:
To make it a bit clearer that this spectrum links directly to reference cited ->

```suggestion
As Christensen shows, at one end of the spectrum you have illustrations that are figurative and just meant to provide a picture of the content under question.
```